pkg/ccn-lite/Makefile

```diff
@@ -7,8 +7,17 @@ PKG_LICENSE=ISC
 
 export RIOT_CFLAGS = ${CFLAGS} ${INCLUDES}
 
-all: git-download
-	cd $(PKG_BUILDDIR)/src && cmake -DCCNL_RIOT=1 -DRIOT_CFLAGS="${RIOT_CFLAGS}" . && make
+all: $(PKG_BUILDDIR)/src/Makefile
+	$(MAKE) -C $(PKG_BUILDDIR)/src && \
 	cp $(PKG_BUILDDIR)/src/lib/libccnl-riot.a ${BINDIR}/ccn-lite.a
 
+$(PKG_BUILDDIR)/src/Makefile: $(PKG_BUILDDIR)/xcompile-toolchain.cmake
+	cd $(PKG_BUILDDIR)/src && \
+	cmake -DCMAKE_TOOLCHAIN_FILE=$(PKG_BUILDDIR)/xcompile-toolchain.cmake \
+		  -DCCNL_RIOT=1 -DRIOT_CFLAGS="${RIOT_CFLAGS}" .
```

Comment:
You could remove the `cd` by using `$(@D)` instead of `.`

Reply:
I'm unsure, because IIRC `cmake` generates its intermediate files where it is called from, which in this case must be `$(PKG_BUILDDIR)/src` otherwise the `make` if above won't work.

Reply:
You are right, I forgot cmake was working this way.